tests/test_tezos_interop.re

```diff
@@ -577,46 +577,102 @@ describe("operation_hash", ({test, _}) => {
   });
 });
 describe("operation forging", ({test, _}) => {
```

Comment:
What about a clear convention for tests, something like 
>When calling _{function}_, successfully _{expected action}_

Because with the actual description, I don't know what is the scope of this test. Maybe the case `when calling forging with bad parameters, fails with this error` is not covered...

Reply:
Please open an issue for this, but those tests are mostly regression test, they're not trying to enforce failure properties, just to ensure that the cases that we actually care are covered here.

Reply:
Created #453